RELEASE/scripts/autoscend/auto_mr2019.ash

```diff
@@ -144,6 +144,9 @@ boolean auto_sausageEatEmUp(int maxToEat)
 			if(mpToBurn > 0)
 				cli_execute("burn " + mpToBurn);
 		}
+
+        acquireMilkOfMagnesiumIfUnused(true);
+        consumeMilkOfMagnesiumIfUnused();
```

Comment:
as previous comment. also (nitpicky) indentation is off here.

Reply:
removed, thanks!